aws/ec2/terraform/security.tf

```diff
@@ -4,7 +4,8 @@ locals {
   // 9090 - Connectors
   // 8080 - Camunda WebUi
   // 443 - HTTPS (e.g. OpenSearch)
-  vpc_ports = [26500, 26501, 26502, 9600, 9090, 8080, 443]
+  // 22 - For SSH from Bastion host
+  vpc_ports = [26500, 26501, 26502, 9600, 9090, 8080, 443, 22]
```

Comment:
isn't the comment above exactly describing which port belongs to which service?

I can make it configurable but again, customers are not limited to just adjust it to their liking.
I don't see module usage = reference arch. It should be used as a base to build on top of.
Modules would be to restrictive.

Reply:
I see where you want to go with this.
Not a fan, as it will require complete rework of the scrips etc., which a customer can do, but I don't intend to have another Helm chart.

Reply:
I'm refactoring it to map(number)

```
variable "ports" {
  type        = map(number)
  default     = {
    ssh = 22
    opensearch_https = 443
    camunda_web_ui = 8080
    camunda_metrics_endpoint = 9600
    connectors_port = 9090
    zeebe_gateway_cluster_port = 26502
    zeebe_gateway_network_port = 26500
    zeebe_broker_network_command_api_port = 26501
  }
  description = "The ports to open for the security groups within the VPC"
}
```

so you have directly "use case" and port number. Can still be used in scripts later if wanted to extract it more easily from tf.

Reply:
added in https://github.com/camunda/camunda-deployment-references/pull/44/commits/76947f0035021c5c54398312489b013c310974b9